intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigDropDown.kt

```diff
@@ -57,9 +61,21 @@ class MirrordConfigDropDown : ComboBoxAction() {
                 return
             }
 
-            initializeConfigFiles(project)
+            if (!::configFiles.isInitialized) {
+                val basePath = project.basePath ?: throw Error("couldn't resolve project path")
+                configFiles = FileBasedIndex.getInstance().getAllKeys(MirrordConfigIndex.key, project).filter {
+                    matches(it, basePath)
+                }.toHashSet()
```

Comment:
How come the code here and in `updateConfigFiles` is different?

Reply:
updateConfigs requries the index with all the keys